templates/vercel/package.json

```diff
@@ -1,13 +1,18 @@
 {
+  "name": "hono",
   "type": "module",
   "scripts": {
-    "start": "vercel dev",
-    "deploy": "vercel"
+    "dev": "vercel dev",
```

Comment:
I ran the `npm run dev` command, but it threw the following error:

```
Vercel CLI 44.6.6
Error: `vercel dev` must not recursively invoke itself. Check the Development Command in the Project Settings or the `dev` script in `package.json`
Error: Learn More: https://vercel.link/recursive-invocation-of-commands
```

<img width="2104" height="376" alt="CleanShot 2025-08-01 at 14 53 00@2x" src="https://github.com/user-attachments/assets/d0430d8f-8e2a-4bd6-bf4e-946bb7d95b7a" />

Reply:
I'm seeing that, as well. We made some late adjustments to how things work.

I've updated those now so things are in working order.